lib/ansible/modules/cloud/ovirt/ovirt_external_provider_info.py

```diff
@@ -44,19 +46,20 @@
     name:
         description:
             - "Name of the external provider, can be used as glob expression."
-extends_documentation_fragment: ovirt_facts
+extends_documentation_fragment: ovirt_info
 '''
 
 EXAMPLES = '''
 # Examples don't contain auth parameter for simplicity,
 # look at ovirt_auth module to see how to reuse authentication:
 
-# Gather facts about all image external providers named C<glance>:
-- ovirt_external_provider_facts:
+# Gather information about all image external providers named C<glance>:
+- ovirt_external_provider_info:
     type: os_image
     name: glance
+  register: result
 - debug:
-    var: ovirt_external_providers
+    var: result.ovirt_external_providers
 '''
 
 RETURN = '''
```

Comment:
From looking at the SDK source, module_utils and module itself, it *looks* like the `ovirt_external_providers` contains a list of dicts, and the content of the dict depends on `type`. The return values described in the docs (`external_host_providers`, `openstack_image_providers`, `openstack_volume_providers`, `openstack_network_providers`) are **probably** not showing up at all. To find out what is really returned, I guess we need someone with access to the API...

I think it's better to leave the docs as-is now, or we replace it by something like
```
ovirt_external_providers:
    description: "List of dictionaries. Content depends on I(type)."
    returned: On success
    type: list
```
The URLs also don't seem to work anymore (at least for me), so maybe removing them will be better? What do you think?

Reply:
That's correct. The content depends on `type`, so your proposed documentation update is correct. The documentation link is only temorarly broken so I would leave it there, because the content is documented there.